prime-router/src/main/resources/db/migration/V61__create_item_table.sql

```diff
@@ -0,0 +1,36 @@
+/*
+This SQL creates the tables of the DB. The Flyway tool applies this migration to create the database
+
+Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+use VARCHAR(63) for names in organization and schema
+
+Copy a version of this comment into the next migration
+*/
+
+/*
+    An item can be a covid test result or any other message (HL7 v2), row (CSV), bundle (FHIR), et. al.
+*/
+CREATE TABLE item (
+    -- Key
+                            id SERIAL PRIMARY KEY,
+
+    -- Value
+                            report_id UUID,
+                            created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    -- Foreign Key
+                            CONSTRAINT report_id
+                                FOREIGN KEY(report_id)
+                                REFERENCES report_file(report_id)
+);
```

Comment:
I don't understand this table. report ID is already a PK in report_file. Why have a table that just have report IDs?